cardano-api/src/Cardano/Api/Orphans.hs

```diff
@@ -278,3 +293,101 @@ instance ToJSON Shelley.RewardType where
 instance ToJSON (SafeHash.SafeHash c a) where
   toJSON = toJSON . SafeHash.extractHash
 
+-----
+
+instance ToJSON Alonzo.ExUnits
+deriving instance FromJSON Alonzo.ExUnits
+
+deriving instance ToJSON Alonzo.Prices
+deriving instance FromJSON Alonzo.Prices
+
+instance ToJSON SBS.ShortByteString where
+  toJSON = Aeson.String . Text.decodeLatin1 . B16.encode . SBS.fromShort
+instance FromJSON SBS.ShortByteString where
```

Comment:
I think we ought not to have to have this instance at all. Why do we needed it? Where is it used?

Reply:
`MemoBytes` is defined like this:

```
data MemoBytes t = Memo {memotype :: !t, memobytes :: ShortByteString}
  deriving (NoThunks) via AllowThunksIn '["memobytes"] (MemoBytes t)
```

Reply:
It is used below in the `MemoBytes` instances.